elementary/config/config.py

```diff
@@ -16,7 +17,7 @@ class Config:
 
     def __init__(self, config_dir: str = DEFAULT_CONFIG_DIR, profiles_dir: str = DEFAULT_PROFILES_DIR,
                  profile_target: str = None, update_bucket_website: bool = None, slack_webhook: str = None,
-                 slack_token: str = None, slack_channel_name: str = None, aws_profile_name: str = None,
+                 slack_token: str = None, slack_channel_name: str = None, slack_timezone: str = None, aws_profile_name: str = None,
```

Comment:
The timezone is part of the alert content and agnostics to the platform it is shared on.
So I think that a better name is just `timezone` 🙂 
Then we will be able to use it to affect other parts of the system if needed!